### PR TITLE
Add a basic cli to lxqt-config-globalkeys

### PR DESCRIPTION
**Pull request by agaida**

### CODE REVIEW

File: config/main.cpp [hunk 1]
```diff
@@ -27,12 +27,23 @@
...
+#include <QCommandLineParser>
 #include "main_window.h"
 
 int main(int argc, char *argv[])
 {
     LXQt::SingleApplication a(argc, argv);
 
+    QCommandLineParser parser;
+    parser.setApplicationDescription(QStringLiteral("LXQt Config Globalkeys "));
+    const QString VERINFO = LXQT_GLOBALKEYS_VERSION \
```
Comment (palinek) [inline config/main.cpp]: Maybe we can do a nano-optimization by usage of `QStringLiteral`:
```diff
index 67a36e4..b280726 100644
--- a/config/main.cpp
+++ b/config/main.cpp
@@ -36,9 +36,9 @@ int main(int argc, char *argv[])
 
     QCommandLineParser parser;
     parser.setApplicationDescription(QStringLiteral("LXQt Config Globalkeys "));
-    const QString VERINFO = LXQT_GLOBALKEYS_VERSION \
+    const QString VERINFO = QStringLiteral(LXQT_GLOBALKEYS_VERSION \
                             "\nliblxqt   " LXQT_VERSION \
-                            "\nQt        " QT_VERSION_STR;
+                            "\nQt        " QT_VERSION_STR);
     a.setApplicationVersion(VERINFO);
     parser.addVersionOption();
     parser.addHelpOption();
```

Comment (palinek) [inline config/main.cpp]: (and this applies to all the PRs)

Comment (agaida) [inline config/main.cpp]: Why not - it it helps us to save two milliseconds at the program start - thats sums up in a year or two :D

Comment (luis-pereira) [inline config/main.cpp]: @agaida I do agree with @palinek . Actually I wrote the same thing on lxqt-admin not knowing this one.

All our new  code should compile witth:
``` CMake
add_definitions(
    -DQT_USE_QSTRINGBUILDER
    -DQT_NO_CAST_FROM_ASCII
    -DQT_NO_CAST_TO_ASCII
    -DQT_NO_URL_CAST_FROM_STRING
    -DQT_NO_CAST_FROM_BYTEARRAY
)
```

Most of our code doesn't and some day it should be addressed. In the meantime, while there are more urgent tasks, we should not worsen the problem.

Comment (agaida) [inline config/main.cpp]: Ok, when such changes make our life less miserable in the sense of future changes we must do this. Full Ack.

**Comment by agaida** [comment]
btw - applied in all PR's

**Pull request merged**